migration/mappings/reach_material.sql

```diff
@@ -19,60 +14,65 @@ SET default_tablespace = '';
 
 SET default_with_oids = false;
 
---
--- TOC entry 584 (class 1259 OID 484607)
--- Name: map_reach_material; Type: TABLE; Schema: sa; Owner: postgres; Tablespace: 
---
+DROP TABLE IF EXISTS map_reach_material;
 
 CREATE TABLE map_reach_material (
-    old integer NOT NULL,
-    new integer
+    old integer,
+    new integer NOT NULL
 );
 
-
 ALTER TABLE sa.map_reach_material OWNER TO postgres;
 
---
--- TOC entry 5386 (class 0 OID 484607)
--- Dependencies: 584
--- Data for Name: map_reach_material; Type: TABLE DATA; Schema: sa; Owner: postgres
---
-
-COPY map_reach_material (old, new) FROM stdin;
-10008	5080
-1	2762
-3	\N
-5	\N
-7	5079
-10	148
-15	5078
-18	\N
-25	5081
-28	\N
-33	153
-38	3016
-10000	5079
-10003	\N
-10004	\N
-10006	5081
-10005	2754
-10007	\N
-10009	\N
-\.
-
-
---
--- TOC entry 5228 (class 2606 OID 484611)
--- Name: map_reach_material_pkey; Type: CONSTRAINT; Schema: sa; Owner: postgres; Tablespace: 
---
-
-ALTER TABLE ONLY map_reach_material
-    ADD CONSTRAINT map_reach_material_pkey PRIMARY KEY (old);
-
-
--- Completed on 2015-06-02 18:12:52 CEST
-
---
--- PostgreSQL database dump complete
---
-
+INSERT INTO map_reach_material (old, new) VALUES
+(10008,	5080), -- SIGE
+(1, 	2754), -- amiante / ciment -> cement
```

Comment:
Here we have some differences, but as said its based on my TB values https://gist.github.com/ponceta/42108b0a381f4d2681aed8d21e67b91b#file-migration_tb2_qgep-sql-L130